plugins/woocommerce-blocks/tests/e2e/utils/editor/editor-utils.page.ts

```diff
@@ -73,7 +73,12 @@ export class Editor extends CoreEditor {
 
 	async revertTemplate( { templateName }: { templateName: string } ) {
 		await this.page.getByPlaceholder( 'Search' ).fill( templateName );
-		await this.page.getByLabel( templateName, { exact: true } ).click();
+		// WP 6.7 issue: https://github.com/WordPress/gutenberg/issues/66585.
+		// Template-parts have aria-label="[object Object]" hence switching
+		// from using label to button.
+		await this.page
+			.getByRole( 'button', { name: templateName, exact: true } )
+			.click();
```

Comment:
This is essentially a switch from using preview (red) to title (green) to access the template
![Screen Shot 2024-10-30 at 11 54 14 AM](https://github.com/user-attachments/assets/4cd1b38b-b895-4bd5-8813-9635bc648895)